pkg/operator/ceph/version/version.go

```diff
@@ -48,12 +48,14 @@ var (
 	Squid = CephVersion{19, 0, 0, 0, ""}
 	// Tentacle ceph version
 	Tentacle = CephVersion{20, 0, 0, 0, ""}
+	// Urchin ceph version
+	Urchin = CephVersion{21, 0, 0, 0, ""}
 
 	// supportedVersions are production-ready versions that rook supports
-	supportedVersions = []CephVersion{Reef, Squid}
+	supportedVersions = []CephVersion{Reef, Squid, Tentacle}
 
 	// unsupportedVersions are possibly Ceph pin-point release that introduced breaking changes and not recommended
-	unsupportedVersions []CephVersion
+	unsupportedVersions = []CephVersion{Urchin}
```

Comment:
Actually no need to add Urchin here. IIRC the unsupportedVersions is only needed if there are older versions that we specifically do not want to support, with a bad ceph bug. We have not used it for some time.

Reply:
Got it